### PR TITLE
Add sort_by_time flag to debug_executor.run method

### PR DESCRIPTION
**Pull request by Icemist**
Sometimes you may want to disable time sorting if you want to compare some outputs with each other.

CC @echuraev

### CODE REVIEW

File: python/tvm/contrib/debugger/debug_executor.py [hunk 1]
```diff
@@ -274,6 +274,7 @@ def run(
         min_repeat_ms=1,
         cooldown_interval_ms=0,
         repeats_to_cooldown=1,
+        sort_by_time=True,
```
Comment (echuraev) [inline python/tvm/contrib/debugger/debug_executor.py:277]: Please, add documentation and then LGTM

Comment (Icemist) [reply python/tvm/contrib/debugger/debug_executor.py:277]: Done. I didn't save it the first time ;)

**Review by echuraev — APPROVED** [review]
LGTM. Thanks!

**Review by tkonolige — APPROVED** [review]
Thanks @Icemist!

**Pull request merged**